File: All Codes/URI_1048.c
#include <stdio.h>
int main()
{
    float a,b;
    int c;
    scanf("%f",&a);
    if (a>=0 && a<=400.00)
    {
        b = a * .15;
        c = 15;
    }
    else if (a>=400.01 && a<=800.00)
    {
        b = a * .12;
        c = 12;
    }
    else if (a>=800.01 && a<=1200.00)
    {
        b = a * .10;
        c = 10;
    }
    else if (a>=1200.01 && a<=2000.00)
    {
        b = a * .07;
        c = 7;
    }
    else if(a>2000)
    {
        b = a * .04;
        c = 4;
    }
    
    printf("Novo salario: %.2f\n",b+a);
    printf("Reajuste ganho: %.2f\n",b);
    printf("Em percentual: %d %%\n",c);

    return 0;
}

File: All Codes/URI_1048.py
# Habibu-R-ahman
# 7th Jan, 2021

a = float(input())
salary = money = percentage = None

if a <= 400:
    salary = a * 1.15
    percentage = 15
elif a <= 800:
    salary = a * 1.12
    percentage = 12
elif a <= 1200:
    salary = a * 1.10
    percentage = 10
elif a <= 2000:
    salary = a * 1.07
    percentage = 7
else:
    salary = a * 1.04
    percentage = 4

print(f"Novo salario: {salary:.2f}")
print(f"Reajuste ganho: {salary - a:.2f}")
print(f"Em percentual: {percentage} %")